java/arcs/jvm/host/JvmHandleManagerProvider.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+package arcs.jvm.host
+
+import arcs.core.host.AbstractArcHost.HandleManagerProvider
+import arcs.core.host.EntityHandleManager
+import arcs.jvm.util.JvmTime
+import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
+
+/**
+ * Provides an [EntityHandleManager] for a JVM-based platform.
+ */
+class JvmHandleManagerProvider(
+    coroutineContext: CoroutineContext = EmptyCoroutineContext
+) : HandleManagerProvider {
+
+    val schedulerProvider = JvmSchedulerProvider(coroutineContext)
+
+    override fun invoke(arcId: String, hostId: String) = EntityHandleManager(
+            arcId = arcId,
+            hostId = hostId,
+            time = JvmTime,
```

Comment:
is StoreManager no longer needed?

Reply:
Right now, each handle manager is creating its own StoreManager (making the assumption that stores exist only per-arc). Stores could be provided to the handlemanager as an argument, if necessary.

Reply:
But wasn't there a bug before where two different arc hosts, or two different arcs, would not see modifications to the same storage key due to them launching a complete storage stack (because they are not using a persistent service)?

Reply:
There was a bug a while back, but that's been fixed. The `DifferentHandleManagerDifferentStores` test variants have been working. 

I may be misremembering, but I thought Shane commented a while back that having a different set of Stores per ArcHost made sense for now, and that the idea of a shared set of stores should be introduced through an explicit "shared storage" mechanism that is yet to be created.

Reply:
That sounds like a good idea, but the problem is, our non-Android tests kind of assume two different EntityHandleManagers in two different arcs or archosts can communicate with the same storage, so I think we'll need to keep that aspect for not until we figure out how to refactor things to support "shared only by explicit"